libraries/Wire/src/Wire.cpp

```diff
@@ -66,9 +66,11 @@ void TwoWire::begin(uint8_t address)
 {
   rxBufferIndex = 0;
   rxBufferLength = 0;
+  rxBuffer = resetBuffer(rxBuffer);
 
   txBufferIndex = 0;
   txBufferLength = 0;
+  txBuffer = resetBuffer(txBuffer);
```

Comment:
ditto

Reply:
Error is raised later if rx/txBuffer is NULL